modules/gapi/test/s11n/gapi_sample_pipelines_s11n.cpp

```diff
@@ -484,4 +489,292 @@ TEST(S11N, Pipeline_GArray_GOpaque_Multinode)
     }
 }
 
+TEST(S11N, Pipeline_Render_NV12)
```

Comment:
The main question is should I separate these 2 tests below into several shorter ones

Reply:
I think it is should
I think it be done as in `modules\gapi\test\render\gapi_render_tests_ocv.cpp`

Reply:
Actually since we already have such tests I prefer to left this as is